### ISSUE TITLE
Simplify branch 'GO:0052305 positive regulation of innate immune response in other organism' 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#17039]
Hello, 

The innate immune response will always be triggered in the host, and never in the symbiont - right ? (not that the symbiont cannot have an innate immune system, but it would be triggered by another synbiont, not by the host). 

I propose to merge some terms to reduce the complexity of this branch: (note: 0 annotations, 0 mappings): 

- 'GO:0052382 induction by organism of innate immune response in other organism involved in symbiotic interaction' -> merge into 'GO:0052390 induction by symbiont of host innate immune response'
- 'GO:0052288 induction by organism of induced systemic resistance in other organism involved in symbiotic interaction' -> merge into 'GO:0052103 induction by symbiont of host induced systemic resistance'
- 'GO:0052289 induction by organism of systemic acquired resistance in other organism involved in symbiotic interaction' -> merge into 'GO:0052104 induction by symbiont of host systemic acquired resistance'
 
@jimhu-tamu @mgiglio99 @cmungall @pmasson55 @nsuvarnaiari @jrr-cpt
Does that seem reasonable ? 

Thanks, Pascale

**Comment by mgiglio99** [comment]
Hi Pascale,

I'm hesitant to do these merges since I can imagine some symbionts with their own innate immunity that might be triggered by host defenses. (Particularly in more complex symbionts like nematodes and other worms.) I don't know of any examples, but I don't have the needed level of expertise. I know they haven't been used in annotations, but I think that's just because the people doing the work on those kinds of organisms aren't making GO annotations. I'd like to do more research or try to get an old PAMGO person to chime in, if I can find one to help.

Michelle

**Comment by ValWood** [comment]
Hi Michelle, 

IMHO it would be better to merge/obsolete and create carefully and correctly defined terms if required.  I have been working with the PHIBase group, who annotate pathogen-host interaction phenotypes, but would also like to provide GO annotation. Navigating GO around this area is really difficult, it seems to have developed independently by different groups in different branches, there is lots of redundancy, little consistency and the term names are largely meaningless to the communities providing annotation. I'm trying to locate terms with expert biologists we are completely lost in the multiple branches. Most phrases commonly used by the fungal pathogen communities are absent from the GO, (although I'm sure the concepts are buried in there somewhere).

It would save a lot of time going forward if we only had to negotiate terms which are clearly  defined, and clearly needed, rather than keeping these odd and badly defined terms just in case.

In my experience,  most of the bad ontology development seems to happen when people build an area of the ontology but are not simultaneously annotating the gene products. 
Otherwise people (experts)  try to shoehorn their favourite things into GO that don't belong (traits - different types of "symbiotic interaction" based on pathogenicity and mutualism , phenotypes, phases, etc). 

If terms were only added when required for annotation, it would circumvent a lot of these issues, and ensure that terms were correctly and meaningfully defined

I would not even merge these, I'd obsolete otherwise we will be lumbered with these unwieldy synonyms that nobody in the community uses.

**Comment by mgiglio99** [comment]
Hi Val,
Speaking just for  PAMGO, the  group was indeed doing annotation while developing terms.  I agree that many of the terms are cumbersome, and I don't  doubt that some have poor definitions. But, it was in an annotation context  that the basic structure was put into place. 
However, in looking over the GO while trying to navigate through this and the other recent related github issue, I have discovered that the main symbiosis term that was kind of the foundation for the PAMGO terms has been redefined. (I must admit I was surprised to see this change.) It looks like a bunch of other stuff in this node changed too - so some of the logical constructs around the original PAMGO terms have been lost . There does seem to be a bit of a jumble now.
(It looks like someone talked about contacting me in that symbiosis issue (#14807) but I don't think they did - at least I can't find any record of it. I'm not complaining , I didn't expect PAMGO terms to be left untouched forever, and I've been out of the GO world for a long time - but I do think we might want to revisit some of those changes. But that will be for another day.)

Regarding the proposed merges above - I see the logic in not keeping terms that aren't being used and which we don't have examples for. However, I worry that doing these types of merges in some areas of this branch and not others runs the risk of creating more inconsistencies.

**Comment by pgaudet** [comment]
Hi @mgiglio99 

@thomaspd and I changed some terms at the top level that was outside the scope of GO, because they represented interactions between organisms (and we don't annotate organisms). Changes are described here: #14807 

This is what the main organization was like: 

<img width="683" alt="image" src="https://user-images.githubusercontent.com/4782928/54433641-abebc800-472c-11e9-87b9-c5ba1b3dffca.png">

I dont think we changed very much in the classes below that. This is some of the children: 

<img width="683" alt="image" src="https://user-images.githubusercontent.com/4782928/54433875-359b9580-472d-11e9-98c6-6ae1922d4466.png">


I'm happy to discuss this on one of the multi-species calls (or any other time). 

Thanks, 
Pascale
